plugins/imageverifier/path_windows.go

```diff
@@ -0,0 +1,25 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package imageverifier
+
+import (
+	"path/filepath"
+
+	"github.com/containerd/containerd/defaults"
+)
+
+var defaultPath = filepath.Join(defaults.DefaultRootDir, "opt")
```

Comment:
This isn't quite specific enough, right?  This appears to be the same raw default as the `opt` plugin, and I think you probably meant to append `image-verifier/bin` to it just like the *nix path did:

```suggestion
var defaultPath = filepath.Join(defaults.DefaultRootDir, "opt", "image-verifier", "bin")
```

Reply:
Fixed, thank you